extension/opampextension/README.md

```diff
@@ -32,6 +32,8 @@ The following settings are optional:
   - `reports_effective_config`: Whether to enable the OpAMP ReportsEffectiveConfig capability. Default is `true`.
 - `agent_description`: Setting that modifies the agent description reported to the OpAMP server.
   - `non_identifying_attributes`: A map of key value pairs that will be added to the [non-identifying attributes](https://github.com/open-telemetry/opamp-spec/blob/main/specification.md#agentdescriptionnon_identifying_attributes) reported to the OpAMP server. If an attribute collides with the default non-identifying attributes that are automatically added, the ones specified here take precedence.
+- `ppid`: An optional process ID to monitor. When this process is no longer running, the extension will emit a fatal error, causing the collector to exit.
```

Comment:
Can we add some docs that explain that this is normally set by the Supervisor (or some other parent process) and is not something users should set manually?

Reply:
Good call, I've added this to the description for the parameter.